### PR TITLE
Fix deepARG test profile and use more stable deepARG db URL

### PR DESCRIPTION
**Pull request by jfy133**
This stabilises the deepARG tests by using the Zenodo uploaded copy of the database.

Note I added the skiup abricate to the test profile to speed up run time (even if it's fast), I also reduced the number of annotation tools to test, to stop us being throttled by Zenodo if we have lots of parallel downloads of the file

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by louperelo — APPROVED** [review]
Looks good. How nice that this test will work now!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -72,31 +72,31 @@ jobs:
...
+          version: "${{ matrix.NXF_VER }}"
 
-  #     - name: Run pipeline with test data (DeepARG workflow)
-  #       run: |
-  #         nextflow run ${GITHUB_WORKSPACE} -profile test_deeparg,docker --outdir ./results ${{ matrix.parameters }}
+      - name: Run pipeline with test data (DeepARG workflow)
+        run: |
+          wget https://zenodo.org/record/8280582/files/deeparg.zip ## download from zenodo due to instability of deepARG server
+          unzip deeparg.zip
+          nextflow run ${GITHUB_WORKSPACE} -profile test_deeparg,docker --outdir ./results ${{ matrix.parameters }} --arg_deeparg_data 'deeparg/deeparg/''
```
Comment (jfy133) [inline .github/workflows/ci.yml:102]:
```suggestion
          nextflow run ${GITHUB_WORKSPACE} -profile test_deeparg,docker --outdir ./results ${{ matrix.parameters }} --arg_deeparg_data 'deeparg/deeparg/'
```

**Review by louperelo — APPROVED** [review]
Looks good to me

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -72,31 +72,31 @@ jobs:
...
+          version: "${{ matrix.NXF_VER }}"
 
-  #     - name: Run pipeline with test data (DeepARG workflow)
-  #       run: |
-  #         nextflow run ${GITHUB_WORKSPACE} -profile test_deeparg,docker --outdir ./results ${{ matrix.parameters }}
+      - name: Run pipeline with test data (DeepARG workflow)
+        run: |
+          wget https://zenodo.org/record/8280582/files/deeparg.zip ## download from zenodo due to instability of deepARG server
+          unzip deeparg.zip
+          nextflow run ${GITHUB_WORKSPACE} -profile test_deeparg,docker --outdir ./results ${{ matrix.parameters }} --arg_deeparg_data 'deeparg/deeparg/'
```
Comment (jfy133) [inline .github/workflows/ci.yml:102]:
```suggestion
          nextflow run ${GITHUB_WORKSPACE} -profile test_deeparg,docker --outdir ./results ${{ matrix.parameters }} --arg_deeparg_data 'deeparg/'
```

**Review by jfy133 — COMMENTED** [review]
Fixing the path

**Pull request merged**